apps/supabase-auth/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Hono } from 'hono';
+import { createClient } from '@supabase/supabase-js';
+
+const app = new Hono();
+
```

Comment:
You'll need to enable CORS to allow other servers to connect (https://hono.dev/middleware/builtin/cors)